### PR TITLE
BXMSPROD-1353 nodejs v16.2.0

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1353

The other chance is to change for the whole RHBA repositories https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1668

### CODE REVIEW

**Review by domhanak — APPROVED** [review]
Thanks for the upgrade. :)

**Comment by Ginxo** [comment]
I would wait for fdb from https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1668 then we will decide what to do

**Comment by Ginxo** [comment]
so https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1668 is not working on appformer-js. I would go with this...

File: appformer-js-monaco/pom.xml [hunk 1]
```diff
@@ -22,6 +22,9 @@
         <artifactId>frontend-maven-plugin</artifactId>
         <configuration>
           <installDirectory>${project.build.directory}</installDirectory>
+          <nodeVersion>v16.2.0</nodeVersion>
+            <npmVersion>7.15.1</npmVersion>
+            <yarnVersion>v1.22.10</yarnVersion>
```
Comment (caponetto) [inline appformer-js-monaco/pom.xml:27]: Hi @Ginxo, thanks for the PR!
Can you please indent these lines? 
Other than this, LGTM.

```suggestion
          <nodeVersion>v16.2.0</nodeVersion>
          <npmVersion>7.15.1</npmVersion>
          <yarnVersion>v1.22.10</yarnVersion>
```

**Comment by Ginxo** [comment]
@tiagobento @mareknovotny can you please merge? latest execution is from https://github.com/kiegroup/kogito-editors-java/pull/62/commits/8bb5a590b82bfb9caee34f21e64db3db5e20fb42 previous checks were all green

**Comment by domhanak** [comment]
Thanks @Ginxo for the effort. This is blocking QE execution of nighty tests. Please address this asap @tiagobento @mareknovotny

**Review by caponetto — APPROVED** [review]

**Review by mareknovotny — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]

**Pull request merged**